### ISSUE TITLE
when script kiddys can't really hack and a 1337 stumbles on it

### ISSUE THREAD

**Issue by copyleftdev** [Lorenzik/TMRB#33]
# Why the "TikTok Mass Report Bot" Code Doesn't Actually Do Anything

## The Gist

So, you've got this code that looks like it's supposed to spam TikTok with reports, right? Well, after giving it the ol' once-over, it's pretty clear this thing's all bark and no bite. Here's the lowdown on why this script is more of a paper tiger than a digital warrior.

### It's Asking for a URL, But Not the Right Kind

- The script's like, "Hey, gimme a URL," but it doesn't tell you it needs something specific to TikTok's report system. It's like asking for a drink and not saying you wanted a soda. Without hitting TikTok's report button directly, it's just shouting into the void.

### It's Not Really Talking to TikTok

- Even when it sends off a request, it's not packing any of the stuff TikTok needs to know what you're upset about. No report details, no target content, nada. It's like sending an empty envelope and expecting a reply.

### Those Console Logs? Yeah, They're Fibbing

- The script pretends it's sending reports by showing messages like "Report sent!" But in reality, there's no chat happening with TikTok. It's the code version of pretending to talk on the phone when it's not even on.

### Proxy Smoxy

- Sure, it uses proxies, which sounds sneaky and cool, but without a real report going through, it's like dressing up for a party that doesn't exist. Nice try, but no cigar.

### Errors Go Poof!

- Any mistakes or hiccups in the script? It's set to ignore them. That's like turning up the music when your car starts making weird noises. Not exactly a fix.

## Bottom Line

This script is like a magician that forgot the magic. It looks fancy, might even get a few oohs and aahs, but at the end of the day, it's not pulling any rabbits out of hats. For anyone diving into code, remember: the proof is in the pudding. Or in this case, the actual code.

**Comment by marevalo13** [comment]
You cooked XD would a script for mass reporting even be possible, need it for certain business reasons.

**Comment by iinja** [comment]
This will be fertile ground for introducing poison into the honey. I have previously reported similar scripts that contain a link to an encrypted external source that is fetched once the command is initiated, using some sort of hidden console that will not show anything invalid to the victim.

**Comment by N0TKZ** [comment]
thank you for posting this 🔥🔥
